src/infrastructure/errors/errorHandler.ts

```diff
@@ -7,7 +7,6 @@ import type { FreeformRecord } from '../../schemas/commonTypes'
 import { isObject, isStandardizedError } from '../typeUtils'
 
 const knownErrors = new Set([
-  'FAST_JWT_MALFORMED',
```

Comment:
Is this the reason for error message change as well?

Reply:
yes, now we are consuming the error that fastify is throwing after catching the underlying error

Reply:
wait, no, actually now we fall into this branch:
```
      const message =
        error.code === 'FST_JWT_AUTHORIZATION_TOKEN_INVALID'
          ? 'Authorization token is invalid'
          : error.message
```

Reply:
so yeah, this is now handled within fastify-thrown error branch, but error message is ours

Reply:
Looks related https://github.com/fastify/fastify-jwt/commit/e24bea573e45d53d3b14df35e032eb8a9399124e

Reply:
yup, looks like it is exactly that change, and it seems to be very intentional